### ISSUE TITLE
Imbalanced confidence during training for different keypoints

### ISSUE THREAD

**Issue by rlrahulkanojia** [vita-epfl/openpifpaf#235]
While training, the confidence of the key points for head and shoulders is improving but the confidence for other keypoints is not increasing that fast in first 12 epochs.with learning rate 1e-4, decay of 0.8, every 100,000 steps.

I am attaching an image for the same. Is this the intended behavior of the loss and confidence
![Screenshot 2020-06-13 at 5 39 21 PM](https://user-images.githubusercontent.com/20882679/84568422-0abf4780-ad9d-11ea-9127-35e4a90e89b9.png)
?

![Selection_329](https://user-images.githubusercontent.com/20882679/84568469-5a057800-ad9d-11ea-95e1-a4b4c309e2b6.png)

**Comment by rlrahulkanojia** [comment]
This is for mobilenet v2 backbone
